pkg/cmd/cli/cmd/status.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/origin/pkg/cmd/cli/describe"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const statusLongDesc = `
+Show a high level overview of the current project.
+
+`
+
+func NewCmdStatus(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Give an overview of the current project",
+		Long:  fmt.Sprintf(statusLongDesc, fullName),
+		Run: func(cmd *cobra.Command, args []string) {
+			client, kclient, err := f.Clients()
+			checkErr(err)
+
+			namespace, err := f.DefaultNamespace()
```

Comment:
Should we also support project as an argument, and use the default if not provided (same approach as `osc project`)?